### ISSUE TITLE
WLan Accesspoint

### ISSUE THREAD

**Issue by xXluki98Xx** [merlinschumacher/Basecamp#59]
Hi, erstmal vielen dank für Basecamp, ist eine sehr schöne und programier Erleichterung.

Nach der Installation meiner Software, bleibt der AccessPoint dauerhaft an, an sich ist das nicht schlimm, allerdings verbindet sich mein Laptop dauerhaft mit dem, da er ein stärkeres Signal hat als mein Router. Ich hab dazu allerdings auch nichts im Netz gefunden.

MfG Lukas

**Comment by EinfachArne** [comment]
Nur zum Verständnis:

Du lädst die Software hoch, verbindest dich mit dem ESP und richtest Basecamp ein. Deine Software läuft dann auch wie gewünscht, aber der Access Point bleibt an?

**Comment by xXluki98Xx** [comment]
Genau. Als ich den Arduino das erste mal mit der Firmware bespielt und mein Programm hoch geladen hatte, funktionierte es auch, nur jetzt nicht mehr. Ich hab auch die Firmware neu geflasht, aber es gibt keine Veränderung.
